src/evm_builtins.rs

```diff
@@ -21,14 +21,18 @@ impl Instructions for EVMInstructions {
         return_vars: &[SMTVariable],
         ssa: &mut SSATracker,
     ) -> String {
-        let single_return = |value: String| {
+        let single_return_with_prefix = |prefix: String, value: String| {
```

Comment:
nit: 
```rust
        let single_return = |value: String, prefix: Option<String>| {
            assert_eq!(return_vars.len(), 1);
            format!(
                "{}\n(define-const {} (_ BitVec 256) {})\n",
                prefix.unwrap_or_default(),
                &return_vars.first().unwrap().name,
                value
            )
        };
```
and no need for `single_return` then

Reply:
I'll remove it altogether for now since it is not needed. But if we need something like this I don't want to change all `single_return` calls to add `None` - or is there a way around it?

Reply:
nope, you are right, you'd have to add `None`